### PR TITLE
Improve MI performance doc

### PR DESCRIPTION
**Pull request by arunans23**
$subject

### CODE REVIEW

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 1]
```diff
@@ -22,16 +22,20 @@ Two key performance metrics were used to measure the performance of each test.
...
-might differ. Similarly when worker pool thread counts are changed, the performance of
-Micro Integrator changes significantly. Hence, for accurate results doing a performance
-test with the exact configuration and scenarios is a must.
+!!! Note
+    
+    All the performance results were done on the default Micro Integrator distribution. The
+    following performance results may become invalid if configurations are changed. For
+    example, the default passthru io_buffer_size is 16KB. Hence the throughput significantly
+    decreases for payload sizes above 16KB. If the buffer size is increased, the throughput
+    might differ. Similarly when worker pool thread counts are changed, the performance of
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:31]:
```suggestion
    might differ. Similarly, when worker pool thread counts are changed, the performance of
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 2]
```diff
@@ -22,16 +22,20 @@ Two key performance metrics were used to measure the performance of each test.
...
+    Micro Integrator changes significantly. Hence, for accurate results doing a performance
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:32]:
```suggestion
    Micro Integrator changes significantly. Therefore, to get accurate results, it is necessary to specify the exact configurations and scenarios used for the performance tests.
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 3]
```diff
@@ -22,16 +22,20 @@ Two key performance metrics were used to measure the performance of each test.
...
+    test with the exact configuration and scenarios is a must.
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:33]:
```suggestion
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 4]
```diff
@@ -22,16 +22,20 @@ Two key performance metrics were used to measure the performance of each test.
...
 
 ## Deployment used for the test
 
+[![MI performance test all-in-one deployment]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:37]:
```suggestion
Given below are the details of the deployment that was used for the performance test this is recorded in this documentation.

[![MI performance test all-in-one deployment]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)]({{base_path}}/assets/img/setup-and-install/performance-test-results/mi_performance_test_all_in_one_deployment.png)
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 5]
```diff
@@ -85,3 +89,43 @@ All scripts used to run the performance tests and analyze results are in the fol
 - [https://github.com/wso2/performance-common](https://github.com/wso2/performance-common)
 
 - [https://github.com/wso2/performance-ei](https://github.com/wso2/performance-ei)
+
+## Results
+
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:94]:
```suggestion
Given below is a summary of the performance result for each of the scenarios used for the tests.
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 6]
```diff
@@ -85,3 +89,43 @@ All scripts used to run the performance tests and analyze results are in the fol
...
+The complete results can be found [here](https://github.com/wso2/micro-integrator/blob/b06581ed31fceaa32c01a03a63a107141a68cb2b/performance/benchmarks/summary.md).
+
+#### Throughput (requests/sec) vs. concurrent users
+
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:98]:
```suggestion
The following results compare the throughput against the number of concurrent users in each scenario.
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md [hunk 7]
```diff
@@ -85,3 +89,43 @@ All scripts used to run the performance tests and analyze results are in the fol
...
+##### CBR Transport Header Proxy
+
+<img src="{{base_path}}/assets/img/setup-and-install/performance-test-results/mi/cbr-transport-header-proxy-tp.png" alt="CBR Transport Header Proxy Throughput" title="CBR Transport Header Proxy Throughput" width="70%" />
+
+##### XSLT Proxy
+
+<img src="{{base_path}}/assets/img/setup-and-install/performance-test-results/mi/xslt-proxy-tp.png" alt="XSLT Proxy Throughput" title="XSLT Proxy Throughput" width="70%" />
+
+#### Average response time (ms) vs. concurrent users
+
```
Comment (nilminiwso2) [inline en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:116]:
```suggestion
The following results compare the average response time of the Micro Integrator server against the number of users in each scenario.
```

File: en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md (hunk 4 above)
Comment (arunans23) [reply en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:37]: `this is recorded in this documentation` or `that is recorded in this documentation`

Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/deployment-best-practices/performance-test-results-mi.md:36]: good catch..

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**